test/src/test/resources/plugins/minJavaVersion-update-center.json

```diff
@@ -0,0 +1,57 @@
+updateCenter.post(
```

Comment:
This is JSONP. Either put it in a file with a `.js` extension, or actually use JSON, which IIRC has been supported for a few years now.

Reply:
I don't disagree, it just feels out of scope given there're many files like these already in the folder. But can fix in a followup if you feel any strongly.